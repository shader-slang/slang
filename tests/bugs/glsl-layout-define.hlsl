//TEST:SIMPLE: -profile vs_5_0

layout(
	binding = UNDEFINED_VK_BINDING,
	set = UNDEFINED_VK_SET) 
Texture2DArray<float4> Float4Texture2DArrays[] : register(t0, space100);

