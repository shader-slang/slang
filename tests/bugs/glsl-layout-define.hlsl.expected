result code = -1
standard error = {
tests/bugs/glsl-layout-define.hlsl(4): error 20001: unexpected identifier, expected integer literal
    binding = UNDEFINED_VK_BINDING,
              ^~~~~~~~~~~~~~~~~~~~
tests/bugs/glsl-layout-define.hlsl(5): error 20001: unexpected identifier, expected integer literal
    set = UNDEFINED_VK_SET)
          ^~~~~~~~~~~~~~~~
}
standard output = {
}
